org.elysium.parent/org.elysium.ui/plugin.xml

```diff
@@ -182,9 +182,15 @@
       <includes>
         <contentExtension pattern="org.eclipse.ui.navigator.resourceContent"/>
         <contentExtension pattern="org.eclipse.ui.navigator.resources.filters.startsWithDot"/>
+        <contentExtension pattern="org.eclipse.ui.navigator.resources.filters.closedProject"/>
         <contentExtension pattern="org.eclipse.ui.navigator.resources.linkHelper"/>
         <contentExtension pattern="org.eclipse.ui.navigator.resources.workingSets"/>
         <contentExtension pattern="org.elysium.ui.filters.NonLilyPondProjects"/>
+        <contentExtension pattern="org.elysium.ui.filters.midi"/>
```

Comment:
I don't clearly see the use case of hiding LilyPond or compiled (midi, pdf, ps) files from the Score Library, which is exactly for browsing these kinds of files. Would you mind keeping only the tempfile filter?

Reply:
My intention for hiding the compiled files is the following: Opening them is possible also via "Show in" on the sources. I generally like a "clean" workspace, i.e. seeing only what I need. So hiding the compile results (potentially numerous) allows to concentrate on the sources to edit (also potentially numerous).

The idea of being able to hide the source is the reverse. If I want to see which scores I have, showing only the pdfs gives a faster overview.

Actually, I'd like to keep all the filters. It's not like you *have* to use them. But to make sure your question about the filters to leave out, could you explicitly enumerate those you'd keep?